colour/io/luts/common.py

```diff
@@ -83,4 +42,4 @@ def path_to_title(path):
     u'ColourCorrect'
     """
 
-    return re.sub('_|-|\\.', ' ', os.path.splitext(os.path.basename(path))[0])
+    return re.sub('_|-|\\.', ' ', os.path.splitext(os.path.basename(path))[0])
```

Comment:
Same here, it is missing the end-of-file return.

Reply:
Noted